packages/vuetify/package.json

```diff
@@ -33,7 +33,7 @@
     "build:dist": "rimraf dist && cross-env NODE_ENV=production webpack --config build/config.js --progress --hide-modules",
     "tsc": "tsc",
     "build:lib": "node build/build-lib.js",
-    "debug:test": "cross-env NODE_ENV=test node --inspect --inspect-brk ./node_modules/jest/bin/jest.js --no-cache -i --verbose",
+    "debug:test": "cross-env NODE_ENV=test node --inspect --inspect-brk ../../node_modules/jest/bin/jest.js --no-cache -i --verbose",
```

Comment:
Does this have anything to do with the PR?

Reply:
I debugged my test with…

Reply:
That shouldn't be in this PR, either there is a problem with running our tests which we are not aware of and should be fixed in a separate issue or there is not problem and this change should be removed from PR

Reply:
→ https://github.com/vuetifyjs/vuetify/pull/7351